app/services/tags/collect_remote_inventory_resources.rb

```diff
@@ -1,36 +1,39 @@
 module Tags
   class CollectRemoteInventoryResources
```

Comment:
Wonder if this class name should include `Topology` in the name. I wasn't immediately sure where this inventory was collected from at first.

Reply:
Is it a future plan to collect from other remote locations or is `Topology` the only one? I kinda feel like it's ok to be ambiguous in the name of this service since you can just look into the class itself and figure out where it's getting its tags from, but if you think there's value of it being `CollectTopologyInventoryResources` then that's probably fine and I can update all of the references to `remote` to `topology` instead.

Reply:
How about `TopologyInventory`? The `tag_resources` `:attr_reader` makes `Resources` in the class name redundant and still clearly defines what is getting collected.

Reply:
Hmm, I think if we go this route we should change the `Tags::CollectLocalOrderResources` class as well to something like `Tags::LocalOrder` and then this would be `Tags::RemoteTopologyInventory`, maybe? Or should we leave "local" and "remote" out of the class name?

Reply:
What about adding another namespace:  `Tags::Topology::CollectRemoteInventoryResources`

If they decide to pull from other resources in the future we can namespace it.

Reply:
Hmm, maybe. I thought you didn't want the word `Resources` in the class name, though?

`Tags::Topology::RemoteInventory.new(args).process.tag_resources`?

Reply:
That sounds great. 👍